.github/workflows/check-lua.yml

```diff
@@ -22,14 +22,13 @@ jobs:
           sparse-checkout: |
             rts/Lua
             rts/Rml/SolLua
-
+      
+      - name: Install Task
```

Comment:
This file needs to revert to original. We bother about reusing code for tasks later.